dbms/src/Storages/Transaction/BackgroundService.cpp

```diff
@@ -36,65 +36,12 @@ BackgroundService::BackgroundService(TMTContext & tmt_)
         },
         false);
 
-    if (!tmt.isBgFlushDisabled())
-    {
-        table_flush_handle = background_pool.addTask([this] {
```

Comment:
`region_mutex`/`regions_to_flush`/`table_flush_handle`/`region_handle` should also be removed
https://github.com/pingcap/tiflash/blob/e1db4677cef8edcb0a573655d659bad0bb451535/dbms/src/Storages/Transaction/BackgroundService.h#L48-L53

Reply:
Check whether `RegionTable::pickRegionToFlush` can be clean or not after we remove these logic

Reply:
okay, thanks